superset/connectors/sqla/views.py

```diff
@@ -167,9 +170,7 @@ class TableModelView(SupersetModelView, DeleteMixin):  # noqa
         'schema': _(
             "Schema, as used only in some databases like Postgres, Redshift "
             "and DB2"),
-        'description': Markup(
-            "Supports <a href='https://daringfireball.net/projects/markdown/'>"
-            "markdown</a>"),
+        'description': _("Supports markdown"),
```

Comment:
There are some special links, such as `https://github.com/d3/d3-format/blob/master/README.md#format`, `<a href='https://docs.python.org/2/library/datetime.html#strftime-strptime-behavior'>`, `http://druid.io/docs/latest/querying/post-aggregations.html` and so on.  

I'm not sure whether I could remove all of them. What's your opinion?